tmt/base.py

```diff
@@ -1874,8 +1874,11 @@ class Clean(tmt.utils.Common):
     def images(self):
         """ Clean images of provision plugins """
         self.info('images', color='blue')
+        successful = True
```

Comment:
@FrNecas I believe we can spare a line here?

```
        for method in tmt.steps.provision.ProvisionPlugin.methods():
            if not method.class_.clean_images(self, self.opt('dry')):
                return False
        return True
```

Reply:
Hm, I think these are different. Even though, currently there is only testcloud which has images, imagine if there was another provision plugin with images which could want to clean them. If the first one failed, then the function would return, not trying to clean the other images. I think that we want to try cleaning all of them